tests/e2e/utils/KubernetesCommandLineToolsExecutor.ts

```diff
@@ -0,0 +1,131 @@
+import { echo, exec, ShellString } from 'shelljs';
+import { KubernetesCommandLineTool, TestConstants } from '../constants/TestConstants';
+import { Logger } from './Logger';
+import { ShellExecutor } from './ShellExecutor';
+
+export class KubernetesCommandLineToolsExecutor extends ShellExecutor {
+    private static container: string;
+    private static pod: string;
+    private readonly namespace: string;
+    private readonly workspaceName: string | undefined;
+    private readonly KUBERNETES_COMMAND_LINE_TOOL: string = TestConstants.TS_API_TEST_KUBERNETES_COMMAND_LINE_TOOL;
+
+    constructor(_workspaceName?: string, _namespace?: string) {
+        super();
+        this.workspaceName = _workspaceName;
+        this.namespace = this.setNamespace(_namespace);
+    }
+
+    get getWorkspaceName(): string {
+        return <string>this.workspaceName;
+    }
+
+    get getNamespace(): string {
+        return this.namespace;
+    }
+
+    loginToOcp(): void {
```

Comment:
In this case, we log in using credentials but another way -  with a token for instance:
` oc login --token=sha256~6FSkPOQ-SsF84WQF046Ow6FkNOtwXcbo8ClzTM3KGmQ --server=https://api.ocp410-developer.crw-qe.com:6443` 
Maybe make sense to add a comment or change the name of the method for better redability

Reply:
Yes, but will we use method with token in tests?

Reply:
Yes we can

Reply:
I added comment above the method